### PR TITLE
Clarifications of TOS

### PR DESCRIPTION
**Pull request by nijel**
## Proposed changes

- remove "own" from name, to avoid impression we require a real name
- remove most of Personal Data in TOS, it provided broader consent than Privacy Policy and that is not really needed
- clarify that payment processors get the data only upon purchase
- add policy on disclosure requests

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

### CODE REVIEW

File: wllegal/templates/legal/documents/tos.html [hunk 1]
```diff
@@ -125,7 +111,7 @@ <h2>{% blocktrans %}Translations{% endblocktrans %}</h2>
 
 <p class="item">{% blocktrans %}The User agrees, upon contributing to a Project, to the license the Project has specified.{% endblocktrans %}</p>
 
-<p class="item">{% blocktrans %}The User agrees to use of own name and e-mail as authorship in the VCS commits. The User understands that this grant is non revocable due to nature of the VCS.{% endblocktrans %}</p>
+<p class="item">{% blocktrans %}The User agrees to use of name and e-mail as authorship in the VCS commits. The User understands that this grant is non revocable due to nature of the VCS.{% endblocktrans %}</p>
```
Comment (orangesunny) [inline wllegal/templates/legal/documents/tos.html:114]: ...non-revocable due to the nature...

File: wllegal/templates/legal/documents/tos.html [hunk 2]
```diff
@@ -98,22 +98,8 @@ <h2>{% blocktrans %}Liability for Damage{% endblocktrans %}</h2>
...
-
-<p class="item">{% blocktrans %}The Provider declares that it will process the Personal Data by electronic means (automatically) via computer programs for which the Provider has secured the right of use, in particular the Service.{% endblocktrans %}</p>
-
-<p class="item">{% blocktrans %}The User’s consent under this Article 5 of the Terms of Service constitutes a free and informed juridical act which concerns the consent of the subject of the data (the User) with processing of the Personal Data.{% endblocktrans %}</p>
-
+{% url 'legal:privacy' as privacy_policy_url %}
+<p class="item">{% blocktrans %}Within the meaning of <a href="{{ url }}">Article 7</a> of the GDPR the User hereby gives consent to the collecting, storage and processing of the Personal Data according to the <a href="{{ privacy_policy_url }}">Privacy Policy</a>.{% endblocktrans %}</p>
```
Comment (orangesunny) [inline wllegal/templates/legal/documents/tos.html:102]: ...gives consent to collecting, storage, and processing...

**Review by orangesunny — CHANGES_REQUESTED** [review]
incorporationg the comments should be enough

**Review by orangesunny — CHANGES_REQUESTED** [review]
just use the semicolon from the only comment and we are goot to go

File: wllegal/templates/legal/documents/privacy.html [hunk 3]
```diff
@@ -57,11 +57,17 @@ <h2>{% blocktrans %}Access to the Personal Data{% endblocktrans %}</h2>
 
 <ul>
     <li>{% blocktrans %}Persons which are contracted for technical assurance of the service.{% endblocktrans %}</li>
-    <li>{% blocktrans %}Payment processors.{% endblocktrans %}</li>
+    <li>{% blocktrans %}Payment processors when purchasing a service from us.{% endblocktrans %}</li>
 </ul>
 
 <p>{% blocktrans %}All Personal Data is stored in the European Union.{% endblocktrans %}</p>
 
+<h2>{% blocktrans %}Disclossure of to the Personal Data{% endblocktrans %}</h2>
```
Comment (phw) [inline wllegal/templates/legal/documents/privacy.html:65]: > Disclossure of to the [...]

Typo? "of" or "to"?

Comment (nijel) [reply wllegal/templates/legal/documents/privacy.html:65]:
```suggestion
<h2>{% blocktrans %}Disclossure of the Personal Data{% endblocktrans %}</h2>
```

File: wllegal/templates/legal/documents/tos.html [hunk 4]
```diff
@@ -98,22 +98,8 @@ <h2>{% blocktrans %}Liability for Damage{% endblocktrans %}</h2>
...
-
-<p class="item">{% blocktrans %}The Provider declares that it will process the Personal Data by electronic means (automatically) via computer programs for which the Provider has secured the right of use, in particular the Service.{% endblocktrans %}</p>
-
-<p class="item">{% blocktrans %}The User’s consent under this Article 5 of the Terms of Service constitutes a free and informed juridical act which concerns the consent of the subject of the data (the User) with processing of the Personal Data.{% endblocktrans %}</p>
-
+{% url 'legal:privacy' as privacy_policy_url %}
+<p class="item">{% blocktrans %}Within the meaning of <a href="{{ url }}">Article 7</a> of the GDPR the User hereby gives consent to collecting, storage, and processing of the Personal Data according to the <a href="{{ privacy_policy_url }}">Privacy Policy</a>.{% endblocktrans %}</p>
```
Comment (nijel) [inline wllegal/templates/legal/documents/tos.html:102]:
```suggestion
<p class="item">{% blocktrans %}Within the meaning of <a href="{{ url }}">Article 7</a> of the GDPR the User hereby gives consent to collecting, storage, and processing of the Personal Data according to the <a href="{{ privacy_url }}">Privacy Policy</a>.{% endblocktrans %}</p>
```

**Pull request merged**